adapter/pokemon_adapter.go

```diff
@@ -3,13 +3,21 @@ package adapter
 import (
 	"errors"
 	"fmt"
+	"sync"
 
 	"github.com/nibble-4bits/ondemand-go-bootcamp/entity"
 )
 
+const (
+	// Max number of goroutines that can be spawned by a worker pool
+	MAX_WORKERS = 20
+)
+
 var (
-	ErrPokemonsNotFound    = errors.New("no pokemons found")
-	ErrPokemonNotFoundByID = errors.New("no pokemon found by ID")
+	ErrPokemonsNotFound      = errors.New("no pokemons found")
+	ErrPokemonNotFoundByID   = errors.New("no pokemon found by ID")
```

Comment:
this's a good convention to specify all the error that you're using in the package